src/components/PageTitle/README.md

```diff
@@ -0,0 +1,43 @@
+A Page Title is a piece of text that describes the current page or screen's content.
+
+You can think of `<PageTitle>` as a combination `<title>` and `<h1>` tag; it is used to describe the content of the entire page/screen available to the user, and should not change unless the user navigates away from the current view. **Every page you create should have one and only one `<PageTitle>` component.**
+
+```jsx
+<PageTitle>My Homepage</PageTitle>
+```
+
+## Usage
+
+Use a `<PageTitle>` to introduce the user to the entirety of the content available in the current view. On a blog, for instance, your index view might use: `<PageTitle>My Posts</PageTitle>` and an individual post would use its post title: `<PageTitle>{post.title}</PageTitle>`.
+
+<!-- TODO for @tofumatt: implement this feature. -->
+When using `<DocumentTitle>`, which uses `react-helmet`, the content inside `<PageTitle>` will be inserted into your page's `<title>` tag. If your `<PageTitle>` contains a link or other content aside from plain text, you should use the `documentTitle` prop to set the content that will be used by `<DocumentTitle>`:
+
+```jsx inside Markdown
```

Comment:
This should go under "Variations" but before I move it ... why would you want to link an h1/page title ever?